components/datatypes/comparisons.schema.json

```diff
@@ -0,0 +1,64 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/comparisons",
```

Comment:
This datatype is never used in this PR. Is this something you expect users to use for future fields of their own? Make sure we only create datatypes if its truly intended to be referenced by many other fields.

Reply:
This will be used in several other locations - we had just included it in this first batch of updates.